src/main/java/uk/gov/hmcts/ccd/CachingConfiguration.java

```diff
@@ -63,6 +63,7 @@ private void configCaches(Config config) {
         config.addMapConfig(newMapConfigWithTtl("caseTypeDefinitionLatestVersionCache", defaultCacheTtl));
         config.addMapConfig(newMapConfigWithTtl("caseRolesCache", defaultCacheTtl));
         config.addMapConfig(newMapConfigWithTtl("jurisdictionCache", jurisdictionCacheTtl));
+        config.addMapConfig(newMapConfigWithTtl("idamUserByIdCache", userCacheTtl)); // Fixed.
```

Comment:
Does the "Fixed" inline comment mean anything?  Some of the `config.addMapConfig(...)` lines have it and some don't.  Wondered if it might be because everyone has copy/pasted an earlier line that happened to have it.

Reply:
Not sure about this @paulridout , removed //Fixed from the code for the new line